weather_finder/weather_finder.py

```diff
@@ -1,85 +1,85 @@
 import requests
 
-API_KEY = '9104e0a25f676b56ac31901d10c5cf0a'
-
-
-def verify_city(json_response):
-    """
-    Takes in response from OWM and checks if city exists
 
-    Arguments:
-       json_response (json): contains response of a request from OWM
-    """
-
-    if json_response['cod'] == '404':
-        print 'City could not be found, please retry'
-	return False
-    else:
-	print 'City found'
-	return True
+API_KEY = '9104e0a25f676b56ac31901d10c5cf0a'
+location = 'London'
+location_url = ('http://api.openweathermap.org/data/2.5/weather?q={}&APPID={}').format(location, API_KEY)
```

Comment:
`WEATHER_API_BASE_URL = 'http://api.openweathermap.org/data/2.5/weather'`
make weather api url in your method.